pandas/io/json/json.py

```diff
@@ -278,8 +278,15 @@ def read_json(path_or_buf=None, orient=None, typ='frame', dtype=True,
 
     typ : type of object to recover (series or frame), default 'frame'
     dtype : boolean or dict, default True
-        If True, infer dtypes, if a dict of column to dtype, then use those,
+        If True, infer dtypes; if a dict of column to dtype, then use those;
```

Comment:
period at the end

Reply:
But that is not the end of the sentence. The end of the sentence is at line 282:
```
dtype : boolean or dict, default True
    If True, infer dtypes; if a dict of column to dtype, then use those;
    if False, then don't infer dtypes at all, applies only to the data.
```